buildSrc/src/main/kotlin/KotlinVersion.kt

```diff
@@ -0,0 +1,14 @@
+@file:JvmName("KotlinVersion")
+
+fun isKotlinVersionAtLeast(kotlinVersion: String, atLeastMajor: Int, atLeastMinor: Int, atLeastPatch: Int): Boolean {
```

Comment:
Please correct me, if we already have a similar function somewhere, I just didn't find it.

Reply:
We have `kotlin.KotlinVersion.CURRENT.isAtLeast()`

Reply:
Fixed